src/python/grpcio_tests/tests/testing/proto/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 gRPC authors.
```

Comment:
Maybe not? I was just following the existing pattern in tests/protoc_plugin/protos.

But when I remove the file, the test fails, so [let's keep it for now and if it can be cleaned up we'll clean it up later](https://github.com/grpc/grpc/issues/11993).